mosquitto/README.md

```diff
@@ -0,0 +1,101 @@
+# Hass.io Core Add-on: Mosquitto broker
+
+Check your current configuration against any Home Assistant version.
+
+![Supports aarch64 Architecture][aarch64-shield] ![Supports amd64 Architecture][amd64-shield] ![Supports armhf Architecture][armhf-shield] ![Supports armv7 Architecture][armv7-shield] ![Supports i386 Architecture][i386-shield]
+
+## About
+
+You can use this add-on to to install Eclipse Mosquitto which is an open source (EPL/EDL licensed) message broker that implements the MQTT protocol. Mosquitto is lightweight and is suitable for use on all devices from low power single board computers to full servers.
+
+## Installation
+
+The installation of this add-on is straightforward and easy to do.
+
+1. Navigate in your Home Assistant frontend to **Hass.io** -> **Add-on Store**.
+2. Find the "Mosquitto broker" add-on and click it.
+3. Click on the "INSTALL" button.
+
+## How to use
+
+The add-on has a couple of options available. For more detailed instructions see [mosquitto]. The basic thing to get the add-on running would be:
+
+1. Start the add-on.
+2. Have some patience and wait a couple of minutes.
+3. Check the add-on log output to see the result.
+
+## Configuration
+
+Add-on configuration:
+
+```json
+{
+  "logins": [],
+  "anonymous": false,
+  "customize": {
+    "active": false,
+    "folder": "mosquitto"
+  },
+  "certfile": "fullchain.pem",
+  "keyfile": "privkey.pem",
+  "require_certificate": false
+}
+```
+
+### Option: `logins`
+
+A list of local users that will be created with username and password. You don’t need do this because you can use Home Assistant users too without any configuration.
```

Comment:
```suggestion
A list of local users that will be created with username and password. You don’t need to do this because you can use Home Assistant users too without any configuration.
```